### DISCUSSION TITLE
Need Help Linking Folate, Methionine, and BH4 Cycles for Deep Learning

### DISCUSSION THREAD

**Discussion by alexmadsen1** [wikipathways/wikipathways-help#63] [Q&A] [answered]
I'm an engineer working on Knowledge-primed neural networks for biologically interpretable deep learning. I've created individual .gpml files for Folate, Methionine, and BH4 cycles, but I am stuck on how to link them at key merge/branch points.

Specifically, I'm not sure how to show:

MTR converting 5-MTHF to THF (Folate Cycle) and Hcy to Met (Methionine Cycle)
DHFR converting BH2 to BH4 (Bh4 Salvage Process) and DHF to THF (Folate Cycle)

Could anyone guide me on properly linking these pathways in my .gpml file / PathVisio? Your expertise will significantly improve the accuracy of my machine-learning models.

I've attached my .gpml file with a .txt extension and an illustrative diagram.

Alex 

[Folate Cycle.txt](https://github.com/wikipathways/wikipathways-help/files/12448969/Folate.Cycle.txt)
![Folate, Methionine,  BH4 Cycle diagram](https://github.com/wikipathways/wikipathways-help/assets/19489773/68d1dd1c-7ee1-49e9-9d6c-81005c0a12e4)
![block diagram with linkages](https://github.com/wikipathways/wikipathways-help/assets/19489773/08a6d44e-303f-4650-8254-89243abe6375)

**Comment by mkutmon** [comment]
Dear Alex,

Is your question about trying to mimic the original image with the rounded lines?
Because the connections you are asking for are already included, right? MTR catalyzes reactions in the different cycles thereby connecting them. You could decide to add colored boxes around each of the subpathways to make them more clearly visible but I think the connections are already there (see highlighted in red below). They are just not round. Is that what you mean?
![image](https://github.com/wikipathways/wikipathways-help/assets/2158343/3ea5435e-7a6f-488a-a2fd-ecf1f2ee78a2)

(Side note: there are a couple of unconnected lines - see green ends of interactions, make sure they are connected to a node properly).

**Reply by alexmadsen1** [reply]
Thank you for getting back to me so quickly. You have identified the relevant connections. My inquiry pertains to encoding these connections as "1" for a single reaction. Specifically, my question centers around how to represent the conversion processes within a single reaction event.

To elaborate, when DHFR facilitates the conversion of BH2 to BH4 during the BH4 salvage process, it simultaneously uses DHF and transforms it into THF within the Folate Cycle. Notably, if the availability of DHF is limited, the BH4 conversion from BH2 cannot take place. This intricate interplay is essential for my neural network model to understand.

**Reply by mkutmon** [reply]
Ah okay. Now, I get it. This is indeed an interesting problem. 
@khanspers, do you  have a suggestion on how to deal with this? My first thought is about a stimulation arrow from DHF to the catalysis interaction of DHFR > towards BH2->BH4 to indicate that DHF is needed for the conversion. Does that make sense?

**Reply by susancoort** [reply]
Dear Alex, We have a one-carbon metabolism pathway in WikiPathways and it partly overlaps with the folate cycle pathway above., https://www.wikipathways.org/pathways/WP3940.html. I would suggest having a look at this pathway and deciding whether you can use the representation of the reactions present. With kind regards, Susan

**Reply by alexmadsen1** [reply]
The one-carbon metabolism pathway in WikiPathways is helpful, and I can use it to check my work. Sadly, I missed it in my search for the Folate Cycle, Methionine Cycle, and BH4 Pathway,  :-( but probably a good thing as I learned a lot in the process of making this.

**Reply by alexmadsen1** [reply]
[mkutmon](https://github.com/mkutmon) . ""Stimulation arrow from DHF to the catalysis interaction of DHFR > towards BH2->BH4 to indicate that DHF is needed for the conversion" That makes sense. I will try that and see if the computer picks it up and can parse it into the network diagram.

**Reply by khanspers** [reply, marked as answer]
@alexmadsen1 @mkutmon : MIM "stimulation" arrow seems reasonable, and maybe even "necessary stimulation"? 

Definitions from the [MIM Specification](https://discover.nci.nih.gov/mim/formal_mim_spec.pdf):
Stimulation: Enhancement of the velocity or extent of a reaction or contingency by the controller entity. 
Necessary stimulation: An interaction where the controller entity is necessary for the controlled interaction to proceed.

**Reply by alexmadsen1** [reply]
Perfect, That is precisely what I needed to Know.

**Comment by KestUrba** [comment]
Hi Alex, I've been interested in the one carbon unit-folate metabolism since 1988! Nice example of mathematical modelling was presented in the article by Reed MC, Thomas RL, Pavisic J, James SJ, Ulrich CM, Nijhout HF. A mathematical model of glutathione metabolism. Theor Biol Med Model. 2008 Apr 28;5:8. doi: 10.1186/1742-4682-5-8. PMID: 18442411; PMCID: PMC2391141. [https://pubmed.ncbi.nlm.nih.gov/18442411/](url). I see the problem  of the lack of POLYAMINE metabolism connected to the Methionine cycle in this Reed's model and Yours drawing. Using math we can simulate narrower or wider systems but evaluating these issues from the biological point of view  we could conclude insufficiency and not systematic approach. It could be fatal in cancer cell studies. Please add an arrow from (Met-)SAM conversion to the polyamines synthesis like this one [https://www.creative-proteomics.com/blog/index.php/brief-introduction-of-one-carbon-metabolism/](url) and this [https://www.mdpi.com/1422-0067/23/3/1919](url) . It is a hot point in cancer development because deficiency of SAM leads to the global hypomethylated DNA

**Comment by alexmadsen1** [comment]
Hi KestUrba,

Thanks for sharing that article; it looks like a valuable resource. I share your long-standing interest in one-carbon metabolism and its implications for mental health. Like you, I've found existing diagrams and models to be frustratingly incomplete. I've combined info from six sources and am now looking to incorporate a seventh.

I'm particularly interested in how this pathway impacts mental health conditions like ADHD, ASD, and their comorbidities (anxiety, depression, bipolar). Many genes linked to these conditions lie along this path.

Your point about the omission of POLYAMINE metabolism in Reed's model and my own work is well-taken. I agree that this could be a crucial gap, I'll be sure to add an arrow for the (Met-)SAM to polyamine synthesis conversion, as you suggested.

Best,

**Reply by KestUrba** [reply]
Thanks. Serine-Folate-Methionine cycle metabolism is an essential key to the epigenetics because of DNA and histone methylation performed by the SAM regulates gene expression and polyamines regulates transcription. People needed exercises to  produce endorfins and improve menthal health ... [https://www.nature.com/articles/s41598-023-37938-9](url)

**Comment by Chris-Evelo** [comment]
While that is true I am always a bit confused by descriptions like this. Yes, you need to make the methyl groups available to make DNA methylation work in the first place, and if there is not enough (or far too much) of a specific methyl donor that might influence genome-wide methylation levels. But typically epigenetic regulation is much more subtle than that, isn't it? It is not like you often drive through a red light because brake manufacturers did a lousy job. So I think we are talking about rather crude epigenetic regulation here, like can also happen on a population level during famine? Epigenetic regulators like MECP2 definitely play an additional role, and in fact, breaking these, also leads to neuronal diseases.

**Reply by KestUrba** [reply]
"But typically epigenetic regulation is much more subtle than that, isn't it?". Indeed I've just described only the first feature of DNA methylation disorders in cancer cells You called it to be crude. There are some forgotten results of methyl deficient diets without any carcinogene  usage for the rat hepatomas initiation obtained by Poirier et al in the 90-ties: 	Poirier LA. The effects of diet, genetics and chemicals on toxicity and aberrant DNA methylation: an introduction. J Nutr. 2002 Aug;132(8 Suppl):2336S-2339S. doi: 10.1093/jn/132.8.2336S. PMID: 12163688 : "by 1990, a chain of causality had been established in experimental carcinogenesis linking dietary methyl deficiency with methyl insufficiency in vivo, as well as with the abnormal methylation of DNA and of specific genes.".  	Pogribny IP, Ross SA, Wise C, Pogribna M, Jones EA, Tryndyak VP, James SJ, Dragan YP, Poirier LA. **Irreversible global DNA hypomethylation as a key step in hepatocarcinogenesis induced by dietary methyl deficiency**. Mutat Res. 2006 Jan 29;593(1-2):80-7. doi: 10.1016/j.mrfmmm.2005.06.028. Epub 2005 Sep 6. PMID: 16144704. It is well known that global DNA hypomethylation and some regions of DNA to be hypermethylated is character to the cancer cell. Moreover, growing embryo cells have globally hypomethylated DNA. So, I obtained an understanding, that DNA methylation disorders and One carbon metabolism is at the root of carcinogenesis in 90-ties and it was stressed  in 2018:   by Adam Rosenzweig, John Blenis, and Ana P. Gomes, Meyer Cancer
Center, NYC. Beyond the Warburg Effect: How Do Cancer Cells Regulate One-Carbon Metabolism?, 2018:
"One-carbon Metabolism at the Root of Carcinogenesis''. [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6103474/](url). Analysis of literature sources revealed much more reasons of DNA disturbances: From my manuscript p,39:
....
The causes of DNA hypermethylation as stated above are:
- Mediation of mutated K-ras induced jun protein (Serra RW. et al. 2014; Leppä S. et al. 1998)
- the inhibitory effect of lnRNA on miRNAs causing demethylation - their "absorption" in the sponge effect or direct repression of demethylation factors TET1 and TGD (Thakur S. Brenner C. 2017; Ratti M. et al. 2020; Morita S. et al. 2013)
- Activation of DNA methylases (Kwon JJ. et al. 2018)
- Changes in isocitrate dehydrogenase (Christensen BC. et al. 2011)
- Effects of viruses (Wang X. et al. )

   Causes of DNA hypomethylation:
- The effect of mutated K-ras on long non-coding RNAs, which, when acting, a) directly inhibits the activity or translation of genes encoding DNA methylases (Sarkar D. et al. 2015) b) rather, "sponges" absorb miRNAs (Ratti M. et al. 2020 ), which should ensure the functioning of DNA methylases
- The effect of mutated K-Ras through the activation of the myc-ODC axis, the mTor complex, with the consequence of the synthesis of polyamines, the activation of which, figuratively speaking, "pumps out" single-carbon fragments from the Methionine cycle and creates a lack of substrate for DNA methylation, leading to a hypomethylated state of DNA (Урба К. 1991 )
- Changes in the activity of methylases DNMT1/3A/3B, their relocalization (Hoffmann MJ, Schulz WA. 2005; Nishiyama A. et al. 2021)
- Changes in TET performance (Nishiyama A. et al. 2021)
- Changes in the synthesis of SAM from methionine due to changes in the enzymes MAT (Frau M. et al. 2013)
- Changes in serine catabolism (Snell K., Weber G. 1986), causing more intensive removal of homocysteine from the methionine cycle, when serine binds to homocysteine (Урба К. 1991)
- Other, unspecified reasons for supplying the Met cycle with single-carbon fragments, causing e.g. "methyl trap" phenomenon (Shane B. Stokstad EL. 1985; Zheng Y, Cantley LC. 2019), sietin and with disorders of vitamin B12 metabolism, disruption of the spare methionine resynthesis pathway (Ouyang Y. et al. 2020; Ozyerli-Goknar E, Bagci-Onder T. 2021; Barekatain, Yasaman et al. 2021) or other monocarbon fragment metabolism disorders (Urba K. 1991). The most interesting for me in the Carcinogenesis model is the global hypomethylation of DNA by the lack of substrate - SAM - s-adenosylmethionine. HOW THESE FACTS COULD BE REPRESENTED in the WIKI PATHWAYS? It is extremly important to understand carcinogenesis and cancer

**Comment by alexmadsen1** [comment]
_"typically epigenetic regulation is much more subtle than that, isn't it? It is not like you often drive through a red light because brake manufacturers did a lousy job."_ That is the key insight, and it touches upon the complexity of how genetic factors contribute to conditions like ASD and ADHD. These conditions are multi-gene, and the cascade-failure nature makes them particularly challenging to diagnose and treat. This is why an accurate and comprehensive map of the network is so important.
